### ISSUE TITLE
Zend OPCache PHP warning

### ISSUE THREAD

**Issue by lumpysimon** [johnbillion/query-monitor#413]
I'm seeing the following PHP warning:

<img width="1093" alt="screen shot 2019-02-22 at 13 05 14" src="https://user-images.githubusercontent.com/625372/53244401-ae657f80-36a2-11e9-98d4-04d24ac7390b.png">

In overview, I see:

`    The Zend OPcache opcode cache is installed but not enabled. External object cache not in use`

I'll email you a link to my `phpinfo` output so you can take a look.

**Comment by johnbillion** [comment]
Thanks. This was reported in the forums too. I'll look into it.

**Comment by johnbillion** [comment]
Fixed in a6a5467 for 3.3.2.

**Comment by lkraav** [comment]
@johnbillion in the case of https://github.com/pressjitsu/pj-object-cache-red we're now shooting PHP notices and warnings all over, because `$cache_data` is initialized inside a conditional, but unconditionally used later.

https://github.com/johnbillion/query-monitor/blob/3.3.2/output/html/overview.php#L32

```
[04-Mar-2019 13:32:32 UTC] PHP Notice:  Undefined variable: cache_data in /wp-content/plugins/query-monitor.git/output/html/overview.php on line 178
[04-Mar-2019 13:32:32 UTC] PHP Notice:  Undefined variable: cache_data in /wp-content/plugins/query-monitor.git/output/html/overview.php on line 191
[04-Mar-2019 13:32:32 UTC] PHP Warning:  array_filter() expects parameter 1 to be array, null given in /wp-content/plugins/query-monitor.git/output/html/overview.php on line 191
[04-Mar-2019 13:32:32 UTC] PHP Notice:  Undefined variable: cache_data in /wp-content/plugins/query-monitor.git/output/html/overview.php on line 208
```
